labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -34,9 +29,7 @@ object ListManipulationExercise01 {
    *  - by using a foldLeft function
    *  - ... etc
    */
-  def lastElementInList[T](l: List[T]): T = {
-    error("fix me")
-  }
+  def lastElementInList[T](l: List[T]): T = l.foldLeft(l.head)( (z, i) => i)
```

Comment:
Interesting. You could do something similar for `foldRight` to get the first element. Also, `.last`.